core/common/src/main/java/alluxio/worker/block/io/LocalFileBlockWriter.java

```diff
@@ -61,11 +61,19 @@ public LocalFileBlockWriter(String path) {
   @Override
   public long append(ByteBuffer inputBuf) {
     long bytesWritten;
+
+    int inputBufLength = inputBuf.limit() - inputBuf.position();
+    MappedByteBuffer outputBuf;
     try {
-      bytesWritten = write(mLocalFileChannel.size(), inputBuf);
+      outputBuf = mLocalFileChannel.map(FileChannel.MapMode.READ_WRITE, mLocalFileChannel.size(),
+          inputBufLength);
     } catch (IOException e) {
-      throw AlluxioRuntimeException.from(e);
+      throw new InternalRuntimeException(e);
```

Comment:
There's other runtime exception (illegalState and illegalArgument)which already covered by from method.